src/js/components/Carousel/__tests__/Carousel-test.tsx

```diff
@@ -1,139 +1,157 @@
 import React from 'react';
-import { render, fireEvent, act } from '@testing-library/react';
+import { render, act, waitFor, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import 'jest-styled-components';
+import '@testing-library/jest-dom';
 
-import { Grommet } from '../../Grommet';
 import { Carousel } from '..';
-import { Image } from '../../Image';
+
+const getSlideOne = () => screen.getByText('Slide One');
+const getSlideTwo = () => screen.getByText('Slide Two');
+const getSlideThree = () => screen.getByText('Slide Three');
 
 describe('Carousel', () => {
   test('basic', () => {
     const { container } = render(
-      <Grommet>
-        <Carousel>
-          <Image src="//v2.grommet.io/assets/IMG_4245.jpg" />
-          <Image src="//v2.grommet.io/assets/IMG_4210.jpg" />
-        </Carousel>
-      </Grommet>,
+      <Carousel>
+        <div>Slide One</div>
+        <div>Slide Two</div>
+      </Carousel>,
     );
-
+    expect(getSlideOne()).toBeVisible();
+    expect(getSlideTwo()).not.toBeVisible();
     expect(container.firstChild).toMatchSnapshot();
```

Comment:
I think we are attempting to migrate to the following:
```suggestion
    expect(asFragment()).toMatchSnapshot();
```

will also need 
```
const { asFragment } = render(...)

```